internal/venice-common/src/main/java/com/linkedin/venice/meta/Store.java

```diff
@@ -341,4 +341,6 @@ static boolean isSystemStore(String storeName) {
   void setIsDavinciHeartbeatReported(boolean isReported);
 
   boolean getIsDavinciHeartbeatReported();
+
+  void updateVersionForDaVinciHeartbeat(int versionNumber, boolean reported);
```

Comment:
We may not need this API, we can just the get the version and set the config.

Reply:
The existing `getVersion()` function returns a read only version so I created this new function to get and update the config similar to `updateVersionStatus`